src/manager.ts

```diff
@@ -1051,10 +1055,8 @@ class Manager<
         options: Pick<ManagerOptions, 'fieldBlackList' | 'fieldWhiteList' | 'pageSize'>
     ): Promise<ESResponse> => {
         try {
-            const request = this._createCustomFilteredQueryRequest(BLANK_ES_REQUEST, options);
-            const response = await this.client.search(removeEmptyArrays(request));
-
-            return response;
+            const request = this.getEsQuery(BLANK_ES_REQUEST, options);
+            return this.client.search(request);
```

Comment:
does this need to be awaited too? it looks like it was removed

Reply:
I understand
```js
function foo() {
  return Promise.resolve('baz')
}
```
and
```js
async function bar() {
  const baz = await Promise.resolve('baz');
  return baz;
}
```
to be equivalent.

What I just noticed that complicates things is the inclusion of a try / catch block. But it looks like the try / catch does nothing anyhow.
I believe
```js
try {
  foo();
catch (e) {
  throw(e)
}
```
to be equivalent to 
```js
foo();
```

Reply:
Agreed, I think we should remove the try-catch since it's not doing anything

Reply:
Lol i've actually done that in a lot of places. There are random catches everywhere....